soapui/src/main/java/com/eviware/soapui/impl/wsdl/support/http/SoapUIMultiThreadedHttpConnectionManager.java

```diff
@@ -248,6 +250,16 @@ public void openConnection(final OperatedClientConnection conn, final HttpHost t
                 boolean last = i == addresses.length - 1;
 
                 Socket sock = sf.createSocket(params);
+                try {
+		    // hostname is required by web server with virtual hosts
+                    PropertyUtils.setProperty(sock, "host", target.getHostName());
+                } catch (IllegalAccessException e) {
+                    e.printStackTrace();

```

Comment:
perhaps you can collapse all these into one catch and delegate to SoapUI.logError( e ) ?